components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -246,7 +446,13 @@ fn migrate_logins(
                     // Weren't successful with local login, if we have a mirror we should
                     // attempt to migrate it and flip the `is_overridden` to false
                     if let Some(mirror_login) = &login.mirror_login {
```

Comment:
This is a weird error case.  We had valid data but failed to insert it.  It seems like something really wrong is happening, like the disk is full or failing.  I don't have any suggestions on how to change the code, but I'll just note that it seems likely that inserts are going to fail in general.

Reply:
I agree, this is really shouldn't happen in the wild without some much bigger issue present but felt like making sure we can at least attempt to prevent data loss (however unlikely) might save us in some small way.